### ISSUE TITLE
Missing runtime dependencies

### ISSUE THREAD

**Issue by BenjaminPelletier** [AlexeyAB/darknet#11]
After successfully compiling darknet following the README, trying to detect with tiny-yolo fails because opencv_core249.lib isn't on the PATH (because the link in the readme just extracts the files to a particular location and doesn't add anything to the path).  Adding the OpenCV lib path to PATH fixes this problem, but then cublas64_80.dll can't be found.  Adding the CUDA path to PATH mysteriously does not fix this problem (see screenshot).  What is the proper way to make sure darknet knows where to find its dependencies at runtime?
![missingruntime](https://cloud.githubusercontent.com/assets/11224086/21996835/ed99f254-dbe0-11e6-80fb-f0edd5d3bd91.png)

**Comment by AlexeyAB** [comment]
@BenjaminPelletier Adding path to `cublas64_80.dll` in Windows system variables should works.

Another way, to put `cublas64_80.dll`, `cudart64_80.dll`, `curand64_80.dll` (and other CUDA-libs which will appear in the error) near with `darknet.exe` file.

**Comment by BenjaminPelletier** [comment]
Thanks again.  Just adding the CUDA and OpenCV paths to PATH didn't seem to work (see screenshot; I added the CUDA path on a previous line and it still gave me the error message).  But, copying the resources locally did seem to work and I can now run tiny-yolo on individual images.  The resources I needed to copy into the same folder as darknet.exe (located in darknet/build/darknet/x64) were:

From opencv_2.4.9/opencv/build/x64/vc12/bin:
opencv_core249.dll
opencv_highgui249.dll

From CUDA/v8.0/bin:
cublas64_80.dll
cudart64_80.dll
curand64_80.dll

That might be helpful to add to the README as well.
